src/push/Permission.js

```diff
@@ -41,13 +41,17 @@ export default class Permission {
     _requestWithCallback(onGranted: () => void, onDenied: () => void) {
```

Comment:
**All of my comments here also apply to the _requestAsPromise() method as well, as the code is structured in the exact same way between methods**